localization/ko/event-sourcing/README.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Event Sourcing
+category: Architectural
+language: ko
+tag:
+
```

Comment:
remove the extra newline

Reply:
Sorry. I'll check it more carefully next time 🤗